src/frontend/src/sass/_form.scss

```diff
@@ -57,9 +57,19 @@
   margin: 0;
 }
 
+.search-container-wrapper {
+  display: flex;
+  align-items: center;
+  margin-top: 15px;
+  width: 100%;
```

Comment:
```suggestion
  width: 100%;
  
  & > .dropdown {
      width: 11em;
    }
```

Reply:
Upon further investigation, the above suggestion may have to be nested a few layers deeper based on the current implementation.